src/pages/AddItem.jsx

```diff
@@ -0,0 +1,142 @@
+import React, { useState } from "react";
+import { Flex } from "../styles/styled";
+import "../App.css";
+import {
+  Label,
+  Input,
+  Title,
+  TextArea,
+  Submit,
+  Form,
+  Tag,
+} from "../styles/styled/Additem";
+import ItemFileInput from "../components/ItemFileInput";
+import CancelLogo from "../assets/ic_X@3x.png";
+import styled from "styled-components";
+
+const INITIAL_VALUE = {
+  imageFile: null,
+  title: "",
+  content: "",
+  price: "",
+  tag: "",
+};
+
+const Cancel = styled.img`
+  width: 22px;
+  height: 24px;
+  cursor: pointer;
+`;
+
+function AddItem() {
+  const [value, setValue] = useState(INITIAL_VALUE);
+  const [tags, setTags] = useState([]);
+  const { imageFile, title, content, price, tag } = value;
+
+  const isValid =
+    title !== "" && content !== "" && price !== "" && tags.length > 0;
+
+  const onChange = (name, value) => {
+    setValue((prev) => ({
+      ...prev,
+      [name]: value,
+    }));
+  };
+
+  const onChangeTags = (value) => {
+    setTags((prev) => [...new Set([...prev, value])]);
+  };
+
+  const handleChange = ({ target }) => {
+    const { name, value } = target;
+    onChange(name, value);
+  };
+
+  const handleEnterTags = (e) => {
+    if (e.key === "Enter") {
+      e.preventDefault();
+      onChangeTags(e.target.value);
+      onChange(e.target.name, "");
+    }
+  };
+
+  const onCancel = (value) => {
```

Comment:
```suggestion
  const onRemoveTag = (value) => {
```
취소보단 태그를 삭제 하는 기능인것같네요